cli/commands/plan.go

```diff
@@ -161,3 +169,96 @@ func HandlePlanSection(app *kingpin.Application) {
 	force.Arg("phase", "Name or UUID of the phase containing the provided step").Required().StringVar(&cmd.Phase)
 	force.Arg("step", "Name or UUID of step to be restarted").Required().StringVar(&cmd.Step)
 }
+
+func toStatusTree(planName string, planJsonBytes []byte) string {
+	optionsJSON, err := client.UnmarshalJSON(planJsonBytes)
+	if err != nil {
+		client.LogMessageAndExit(fmt.Sprintf("Failed to parse JSON in plan response: %s", err))
+	}
+	var buf bytes.Buffer
+
+	planStatus, ok := optionsJSON["status"]
+	if !ok {
+		planStatus = "<UNKNOWN>"
+	}
+	buf.WriteString(fmt.Sprintf("%s (%s)\n", planName, planStatus))
+
+	phases, ok := optionsJSON["phases"].([]interface{})
+	if ok {
+		for i, rawPhase := range phases {
+			appendPhase(&buf, rawPhase, i == len(phases) - 1)
+		}
+	}
+
+	errors, ok := optionsJSON["errors"].([]interface{})
+	if ok && len(errors) > 0 {
+		buf.WriteString("\nErrors:\n")
```

Comment:
The scope doc calls for Errors to be output as a list within square brackets, e.g.:
```
Error: ['New config's PodSpec named 'world' has 1 tasks, expected >=2
    tasks', ...]
```

I don't have a strong opinion one way or another.